manifests/attr.pp

```diff
@@ -1,9 +1,10 @@
 # Install FreeRADIUS config snippets
 define freeradius::attr (
   String $source,
-  Freeradius::Ensure $ensure = present,
-  Optional[String] $key      = 'User-Name',
-  Optional[String] $prefix   = 'filter',
+  Freeradius::Ensure $ensure           = present,
+  Optional[String] $key                = 'User-Name',
+  Optional[String] $prefix             = 'filter',
+  Optional[Enum['yes', 'no']] $relaxed = undef,
```

Comment:
I forget we have this `Freeradius::Boolean` type - which is `Enum['yes','no']` :-)

```suggestion
  Optional[Freeradius::Boolean] $relaxed = undef,
```